docs/data/material/getting-started/templates/crud-dashboard/data/employees.ts

```diff
@@ -0,0 +1,201 @@
+import type { GridFilterModel, GridPaginationModel, GridSortModel } from '@mui/x-data-grid';
+
+type EmployeeRole = 'Market' | 'Finance' | 'Development';
+
+export interface Employee {
+  id: number;
+  name: string;
+  age: number;
+  joinDate: string;
+  role: EmployeeRole;
+  isFullTime: boolean;
+}
+
+const INITIAL_EMPLOYEES_STORE: Employee[] = [
+  {
+    id: 1,
+    name: 'Edward Perry',
+    age: 25,
+    joinDate: new Date().toISOString(),
```

Comment:
Looks like this caused [unstable screenshots](https://app.argos-ci.com/mui/material-ui/builds/39978/178329167)